src/uikit.cs

```diff
@@ -127,6 +143,43 @@ interface NSAttributedStringAttachmentConveniences {
 	}
 #endif
 
+	[NoWatch, NoTV, iOS (10,0)]
+	[DisableDefaultCtor]
+	[Abstract] // abstract class that should not be used directly
+	[BaseType (typeof (NSObject))]
+	interface UIFeedbackGenerator {
+
+		[Export ("prepare")]
+		void Prepare ();
+	}
+
+	[NoWatch, NoTV, iOS (10,0)]
+	[BaseType (typeof(UIFeedbackGenerator))]
+	interface UIImpactFeedbackGenerator {
+
+		[Export ("initWithStyle:")]
+		IntPtr Constructor (UIImpactFeedbackStyle style);
+
+		[Export ("impactOccurred")]
+		void ImpactOccurred ();
+	}
+
+	[NoWatch, NoTV, iOS (10,0)]
+	[BaseType (typeof(UIFeedbackGenerator))]
+	interface UINotificationFeedbackGenerator {
+
+		[Export ("notificationOccurred:")]
+		void NotificationOccurred (UINotificationFeedbackType notificationType);
+	}
+
+	[NoWatch, NoTV, iOS (10,0)]
+	[BaseType (typeof(UIFeedbackGenerator))]
+	interface UISelectionFeedbackGenerator {
```

Comment:
mmm there is not initWithxxx: in the base ctor and docs do not have any info about it